app/main/forms/brief_forms.py

```diff
@@ -0,0 +1,91 @@
+from flask_wtf import Form
+from wtforms import IntegerField, SelectMultipleField
+from wtforms.validators import NumberRange
+
+
+class BriefSearchForm(Form):
```

Comment:
Yeah, I'm with @risicle on this one. 

We did a bit of digging through the `/g-cloud/search` endpoint in the search API this morning and there's a bunch of hard-coded `g-cloud-6` stuff in there where it's looking for a manifest of search filters.   Seems like writing this in a more robust way should also mean rethinking where we keep our manifest for which search filters to use, and that's getting a bit beyond what this story really needs to be.

It would be neat to seed the WTForm with content that lives in `digitalmarketplace-frameworks` (instead of hardcoding it into the form), but it seems like if we're going with Rob's stuff right now on the argument that the `g-cloud` search is dissimilar, we shouldn't really be trying to pull the stuff from the content loader into here until we're more aligned on how to merge everything.  At the moment, it all looks pretty straightforward to me.

As a final point, I remember that when David and I were working on the supplier submissions framework submission, we also weren't clear on whether or not to use WTF forms.  In our case, we ended up doing [WTF forms for entering the signer names and the final "I have the authority" form](https://github.com/alphagov/digitalmarketplace-supplier-frontend/blob/master/app/main/forms/frameworks.py) but not for the file uploading bit because there was a bunch of file uploading code that we reused.  So, while we've got the submissions bit pretty much nailed down, I think there's still a bunch of ambiguity around building other forms that don't (or don't have to) involve the content loader.